project-base/storefront/components/Forms/validationRules.ts

```diff
@@ -170,3 +170,26 @@ export const validateSecondPassword = (t: Translate): BaseSchema => {
         )
         .oneOf([Yup.ref('passwordFirst'), null], t('Passwords must match'));
 };
+
+export const validateNewPassword = (t: Translate): BaseSchema => {
+    return Yup.string()
+        .required(t('Please enter new password'))
+        .min(
+            VALIDATION_CONSTANTS.passwordMinLength,
+            t('Password must be at least {{ count }} characters long', {
+                count: VALIDATION_CONSTANTS.passwordMinLength,
+            }),
+        );
+};
+
+export const validateNewPasswordAgain = (t: Translate): BaseSchema => {
+    return Yup.string()
+        .required(t('Please enter new password again'))
+        .min(
+            VALIDATION_CONSTANTS.passwordMinLength,
+            t('Password must be at least {{ count }} characters long', {
+                count: VALIDATION_CONSTANTS.passwordMinLength,
+            }),
+        )
+        .oneOf([Yup.ref('newPassword'), null], t('Passwords must match'));
```

Comment:
Seems like you've missed the `otherwise: Yup.string(),` case from the original code. Was that on purpose?